packages/realm-server/scripts/start-base-root.sh

```diff
@@ -1,6 +1,21 @@
 #! /bin/sh
+check_postgres_ready() {
+  docker exec boxel-pg pg_isready -U postgres >/dev/null 2>&1
+}
+# remove this check after the feature flag is removed
+if [ -n "$PG_INDEXER" ]; then
+  while ! check_postgres_ready; do
+    printf '.'
+    sleep 1
```

Comment:
Should there be a timeout like I’m working on adding [here](https://github.com/cardstack/boxel/blob/825d4511aa504d04c0256ddd6e52b37cf2ee8c69/packages/matrix/scripts/register-realm-users.sh#L12)?

Reply:
Sure